keps/2076-kueuectl/README.md

```diff
@@ -339,7 +340,45 @@ so that the users won't have to remember to switch the command to kubectl.
 * `describe workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
 * `edit workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
 * `patch workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
-* `delete workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
+* `delete clusterqueue|cq|localqueue|lq|resourceflavor|rf`
+
+### Delete Workload
+
+Motivation:
+The pass-through delete workload command doesn't actually delete the Workload but 
+instead requeues it. This can confuse users, as the Workload remains listed even 
+after they attempt to delete it, due to its recreation.
```

Comment:
```suggestion
The pass-through delete workload command deletes the Workload API object, which is later
recreated leading to its job to be re-queued. This can confuse users, as the recreated Workload
has the same name giving the impression that the `delete workload` command did not succeed.
```

Reply:
Fixed